app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -174,33 +174,33 @@ export const APITracingTypeEnum = Enum.make({
 
 export type APITracingType = $Keys<typeof APITracingTypeEnum>;
 
-export type APITaskTypeType = {
+export type APITaskType = {
   +id: string,
   +summary: string,
   +description: string,
   +teamId: string,
   +teamName: string,
-  +settings: APISettingsType,
+  +settings: APISettings,
 };
 
-export type TaskStatusType = { +open: number, +active: number, +finished: number };
+export type TaskStatus = { +open: number, +active: number, +finished: number };
 
 type APIScriptTypeBase = {
```

Comment:
Those `...TypeBase` types were of course not renamed, but I think it's fine to do this gradually when seeing it in the code :)